.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  test_e2e:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:10-browsers
+      
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Set up orient-express server.
+      # get last commit hash.
+      - run: curl --silent -u $GITHUB_USER:GITHUB_PASSWORD https://api.github.com/repos/lesfieres/orient-express/commits | head -3 | tail -1 | cut -c 13-52 > orient-express-last-commit-hash
+      - run: cat orient-express-last-commit-hash
+
+      # Restore cache.
+      # TODO: Set up cache for orient-express.
+      # - restore_cache: 
+      #    keys: 
+      #      - v1-orient-express-{{ checksum "orient-express-last-commit-hash" }}
+
+      # Clone and prepare .env
+      - run: git clone https://github.com/lesfieres/orient-express.git
+      - run:
+          name: "Setup orient-express"
+          command: |
+            cd orient-express
+            npm install
+            echo -n 'GOODREADS_KEY=' >> .env
+            echo $GOODREADS_KEY >> .env
+            echo -n 'GOODREADS_SECRET=' >> .env
+            echo $GOODREADS_SECRET >> .env
+            echo -n 'OMBD_KEY=' >> .env
+            echo $OMBD_KEY >> .env
```

Comment:
Would love to have a more clever way to solve this, LGMT thought.

Reply:
Sure, marcP is working on it on his branch :)